### PR TITLE
[11.0][IMP] updates for partenr_banks_to_show method

### PR DESCRIPTION
**Pull request by i-vyshnevska**
improve some logic on invoicing and bank searching

### CODE REVIEW

File: l10n_ch_base_bank/models/bank.py [hunk 1]
```diff
@@ -369,10 +378,9 @@ def onchange_acc_number_set_swiss_bank(self):
 
     @api.onchange('ccp')
     def onchange_ccp_set_acc_number(self):
-        """If ccp changes and it's a postal bank update acc_number to ccp
-        we don't want make acc_number as computed to have possibility set it
-        manually and also avoid to shadow other logic on acc_number if exist
-       """
+        # If ccp changes and it's a postal bank update acc_number to ccp
+        # we don't want make acc_number as computed to have possibility set it
+        # manually and also avoid to shadow other logic on acc_number if exist
```
Comment (yvaucher) [inline l10n_ch_base_bank/models/bank.py]: Why are the docstring replaced by comments?

File: l10n_ch_base_bank/tests/test_search_invoice.py [hunk 2]
```diff
@@ -29,14 +29,15 @@ def setUp(self):
             'type': 'bank',
             'code': 'BNK42',
             'bank_id': bank.id,
-            'bank_acc_number': '01-1234-1',
+            'bank_acc_number': '10-8060-7',
```
Comment (yvaucher) [inline l10n_ch_base_bank/tests/test_search_invoice.py]: Why do this value change?

**Review by yvaucher — CHANGES_REQUESTED** [review]
2 questions

Plus I spotted a typo in the commit message `partenr`

File: l10n_ch_base_bank/models/invoice.py [hunk 3]
```diff
@@ -1,7 +1,6 @@
-# Copyright 2012 Nicolas Bessi (Camptocamp SA)
-# Copyright 2015 Yannick Vaucher (Camptocamp SA)
+# Copyright 2012-2019 Camptocamp
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-from odoo import models, api, _
+from odoo import models, api, fields, _
```
Comment (yvaucher) [inline l10n_ch_base_bank/models/invoice.py]: you can remove `fields` from the imports

File: l10n_ch_base_bank/tests/test_search_invoice.py (hunk 2 above)
Comment (i-vyshnevska) [reply l10n_ch_base_bank/tests/test_search_invoice.py]: bank_acc_number should be unique, when I'd relapsed on create it's start failing from which point I don't remeber

Comment (i-vyshnevska) [reply l10n_ch_base_bank/tests/test_search_invoice.py]: ok, I restored the initials numbers (before my fix)

**Review by yvaucher — APPROVED** [review]

**Pull request merged**